### ISSUE TITLE
Gender proportions in the R community

### ISSUE THREAD

**Issue by benmarwick** [ropensci/unconf#13]
This is more of a data science project than a software development issue, but rOpenSci is notable for promoting gender equity, so it seems relevant. The goal would be to identify specific areas in the R user and developer communities to promote gender diversity and counter bias. The general approach would be to use the [gender package](http://cran.r-project.org/web/packages/gender/index.html) to analyse various data sources on contributions to the R community to investigate patterns in proportions of female to male contributors.  This is currently a topical issue, as we can see from a summary of recent conversations:
- there was a lively [r-help thread](https://stat.ethz.ch/pipermail/r-help/2014-November/thread.html#423632) on this last year, 
- a related bit of [activity of twitter](https://twitter.com/scottkosty/status/449933971644633088) about gender ratios on r-help
- there was a "Conversation about R’s Gender Gap: The useR! her Panel" at the [2014 UseR meeting](http://user2014.stat.ucla.edu/)
- @jennybc, in conversation with @hadley, @earino and @ledell, [noted on twitter](https://twitter.com/JennyBryan/status/544970521348034561) that there is only one female name in the 90 contributors listed on the [R Project Contributors](http://www.r-project.org/contributors.html) page
- more recently @dicook has done some work with her students on this, and I've made a [sketch](https://gist.github.com/benmarwick/6bc3c6d0f09e3699eed9) 
- probably others I don't know about, eg. [Hadley says](https://twitter.com/hadleywickham/status/544937382404177921) 'we're aware of the problem and there are plans to fix it' 

Here are a few specific questions that could be considered:
- It's a safe bet that males are a higher proportion overall, but is this changing over time? For example, what are the proportions among package maintainers (9-15% female by some estimates, and a slight but non-significant decline in recent years)? Those data are nicely structured and easy to access, for example at http://www.rdocumentation.org/
- Is the proportion of female R developers high enough to make a case that R core should include at least one female? Who do the data suggest this should be?
- Are there some areas of R development where females have greater representation? For example are there more female package maintainers on github or bioconductor than CRAN?
- Is there are difference in the way bug reports from male and female posted to the R-base bugtracker are handled? Do they equally lead to fixes?
- Is there are difference in name order for males and females amongst packages with multiple maintainers?

**Comment by ctb** [comment]
You could also take a look through mailing archives to estimate hostility quotients and the like for discussions, because that turns off new contributors (including women) and sets the tone for the whole project. Might get a bit personal, but could still be valuable; and could extend well beyond the R community in impact.

**Comment by benmarwick** [comment]
Yes, good idea, reminds me of @treycausey's interesting analysis of r-help:  
http://badhessian.org/2013/04/has-r-help-gotten-meaner-over-time-and-what-does-mancur-olson-have-to-say-about-it/

**Comment by dicook** [comment]
I wonder if the ggplot2 mailing list might make a good comparison, because I think there are a few women that regularly answer questions there. And I think it is politer. I may be wrong.

From the little black box...
